packages/api/src/constants/onfido.ts

```diff
@@ -0,0 +1,350 @@
+import React from 'react';
+
+export const ROOT_CLASS = 'manual-poi-details';
+
+export const DOCUMENT_TYPES = {
+    NATIONAL_IDENTITY_CARD: 'national_identity_card',
+    NIMC_SLIP: 'nimc_slip',
+    PASSPORT: 'passport',
+    DRIVING_LICENCE: 'driving_licence',
+    BIRTH_CERTIFICATE: 'birth_certificate',
+    SELFIE_WITH_ID: 'selfie_with_id',
+    OTHER: 'other',
+};
+
+const PAGE_TYPE = {
+    FRONT: 'front',
+    BACK: 'back',
+    PHOTO: 'photo',
+};
+
+export const SELFIE_DOCUMENT = {
+    document_type: DOCUMENT_TYPES.SELFIE_WITH_ID,
+    pageType: PAGE_TYPE.PHOTO,
+    name: 'selfie_with_id',
+    icon: 'IcSelfie',
+    info: 'Upload your selfie.',
+};
+
+export const date_field = {
+    name: 'expiry_date',
+    label: 'Expiry date',
+    type: 'date',
+    required: true,
+};
+
+export const ONFIDO_PHRASES = {
```

Comment:
just idea for future considerations/future PRs, as per https://en.wikipedia.org/wiki/Zen_of_Python
"Flat is better than nested"

`{
    "country_select_alert_dropdown_country_not_found": "Country not found",
    "country_select_alert_another_doc": "Documents from that country are not currently supported — <fallback>try another document type</fallback>",
    "country_select_button_primary": "Submit document",
    "country_select_search_accessibility": "Select country",
    "country_select_search_input_placeholder": "e.g. United States",
    "country_select_search_label": "Search for country",
    "country_select_title": "Select issuing country",
    "cross_device_checklist_button_primary": "Submit verification",
    "cross_device_checklist_info": "Tips"
    `